tests/integration/ovs_test.py

```diff
@@ -387,6 +387,36 @@ def test_ovs_vlan_access_tag():
     assertlib.assert_absent(PORT1)
 
 
+@pytest.mark.tier1
+def test_ovs_vlan_trunks():
```

Comment:
This test won't be available in all NM version. We need to skip it when NM version is 1.41 (we do not care about the micro version as stable won't use 1.41 at all) or greater.

```
@pytest.mark.skipif(nm_major_minor_version() < 1.41, reason="OVS VLAN trunks was not supported in NM")
```

Add this after the `@pytest.mark.tier1` decorator so CI will skip it when using NM stable :-)